src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -105,7 +105,7 @@ private void changeSystemReturnDate(DateTime systemReturnDate) {
     write(representation, SYSTEM_RETURN_DATE, systemReturnDate);
   }
 
-  private void changeAction(String action) {
+  public void changeAction(String action) {
```

Comment:
For overridden renewals, these methods were used internally within a specific method. Why was this done different for check out?

Is it because a check out is the initial operation, and it didn't make sense to have a method named `checkOut`?

Reply:
Yes, that is the main reason.